tests/smoke/notion_fetch.rs

```diff
@@ -0,0 +1,31 @@
+use hamcrest2::core::Matcher;
+use test_support::matchers::execs;
+use support::sandbox::sandbox;
+
+#[test]
+fn fetch_node() {
+    let s = sandbox()
+        .build();
+
+    assert_that!(
+        s.notion("fetch node 10.4.1"),
+        execs()
+            .with_status(0)
+    );
+    assert_eq!(s.node_version_is_fetched("10.4.1"), true);
```

Comment:
Can we get comments/documentation on why the node version is different in each of the smoke tests?  And should the yarn versions be different as well?

Reply:
I am using different versions of node and yarn for these tests just in case the environment is not cleaned up correctly, to help with test isolation. I'll add some comments about that.